src/accounts/accounts.service.ts

```diff
@@ -223,11 +226,13 @@ export class AccountsService {
     //locked is false, not enable and password is null
 
     if (activeAccount.role !== RoleEnum.USER)
-      throw new Error('Account must be user account to be actived');
-    if (activeAccount.isLocked) throw new Error('Account has been locked');
-    if (activeAccount.isEnabled) throw new Error('Account has been actived');
+      errorList.push('Account must be user account to be actived');
+    if (activeAccount.isLocked) errorList.push('Account has been locked');
+    if (activeAccount.isEnabled) errorList.push('Account has been actived');
     if (activeAccount.password)
-      throw new Error('Account password has been generated');
+      errorList.push('Account password has been generated');
```

Comment:
U don't need to check this because when u have check isEnabled mean it has a password